src/page/blog/article-page.jsx

```diff
@@ -1,125 +1,250 @@
+              }
+            />
+            <Form.TextInput
+              className=" flex font-bold font-arial flex-col py-3"
+              labelText={"Slops"}
+              placeholder={"Add topical slops"}
+              value={formState.movies.join(", ")}
+              onChange={(e) =>
+                setFormState({
+                  ...formState,
+                  movies: e.target.value.split(",").map((item) => item.trim()),
+                })
+              }
+              isValid={checkValidity(formState.movies.join(", "))}
+            />
+            <Form.TextInput
+              className=" flex font-bold font-arial flex-col py-3"
+              labelText={"Keywords"}
+              placeholder={"Add topical keywords"}
+              value={formState.keywords.join(", ")}
+              onChange={(e) =>
+                setFormState({
+                  ...formState,
+                  keywords: e.target.value
+                    .split(",")
+                    .map((item) => item.trim()),
+                })
+              }
+              isValid={checkValidity(formState.keywords.join(", "))}
+            />
+          </Form>
+          <div className="self-center mt-32 h-[49px] min-w-[224px] md:absolute md:bottom-0 md:left-0 md:right-0 md:top-96 xs:absolute xs:bottom-0 xs:left-0 xs:right-0 xs:top-80">
+            <Form onSubmit={onDraft}>
+              <Form.Submit
+                className="font-bold font-arial bg-white text-lg/4 text-black w-full border border-black py-4 px-4"
+                title={"Save to drafts"}
+                onSubmit={onDraft}
+              />
+            </Form>
+
+            <Form onSubmit={onPublish}>
+              <Form.Submit
+                className="font-bold font-arial bg-yellow-400  text-lg/4 text-black w-full border border-black py-4 px-4"
+                title={"Publish!"}
+                disabled={false}
+              />
+            </Form>
+          </div>
+        </div>
+      ) : (
+        // the ticket that will turn the following success display
+        // into a component doesn't seem to be done yet, so for now the
+        //  markup is just hard coded in
+
+        <div className="max-w-[1440px] mx-auto ">
+          <div className="flex flex-col justify-center items-center xs:px-5 sm:px-5">
+            <h1 className="mb-40 mt-10 Arial-NarrowBold text-5xl">SLOP BLOG</h1>
+            <p className="max-w-[627px] xs:text-sm sm:text-center md:text-center lg:text-center">
+              Thanks for submitting a slop to our platform, dear goblin!
+            </p>
+            <p className="max-w-[632px] xs:text-sm ">
+              Our team of professional slop goblins will review your submission
+              and publish it, if your slop is actually sloppy, and doesn't
+              repeat movies already published here.
+            </p>
+            <div className="mt-40 xs:mt-16 ">
+              <Button
```

Comment:
Should have a second button similar to link where it takes the user back to the articles page if they don't want to post another article.
![image](https://github.com/jahorwitz/slopopedia/assets/110426130/34b22b42-e28c-4e6c-8463-55fd8574acc7)
something similar to this underneath the button where if clicked it goes back to articles page.